src/main/java/frc/robot/DistanceRange.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot;
+
+import java.util.HashMap;
+
+import frc.robot.AutonomousPath.VisionPathName;
+
+public class DistanceRange 
+  implements DistanceRangeInterface{
+
+    HashMap<String, Double> distanceRanges;
```

Comment:
missing private designation: you probably dont want package protected ( the default). also its more readable to initiaize on this line vs in the constructor